src/skydb.ts

```diff
@@ -283,6 +289,100 @@ export async function setDataLink(
   await this.registry.setEntry(privateKey, entry, setEntryOpts);
 }
 
+/**
+ * Gets the raw registry entry data at the given public key and data key.
+ *
+ * @param this - SkynetClient
+ * @param publicKey - The user public key.
+ * @param dataKey - The data key.
+ * @param [customOptions] - Additional settings that can optionally be set.
+ * @returns - The entry data.
+ */
+export async function getEntryData(
+  this: SkynetClient,
+  publicKey: string,
+  dataKey: string,
+  customOptions?: CustomGetEntryOptions
+): Promise<EntryData> {
+  validateHexString("publicKey", publicKey, "parameter");
+  validateString("dataKey", dataKey, "parameter");
+  validateOptionalObject("customOptions", customOptions, "parameter", DEFAULT_GET_ENTRY_OPTIONS);
+
+  const opts = {
+    ...DEFAULT_GET_ENTRY_OPTIONS,
+    ...this.customOptions,
+    ...customOptions,
+  };
+
+  const { entry } = await this.registry.getEntry(publicKey, dataKey, opts);
+  if (!entry) {
+    return { data: null };
+  }
+  return { data: entry.data };
```

Comment:
Maybe you can compare `entry.data` with `new Uint8Array(RAW_SKYLINK_SIZE)` here and return `null` if they match. I think that would be more correct. I'm not sure though, I think you can also argue it's fine the way it is. I prefer returning `null` though for deleted entries.

Reply:
Ah, there's the potential bit of magic I was thinking about when replying to https://github.com/SkynetLabs/skynet-js/pull/298/files#r715401542. :) I don't mind very much but we would need to seriously document this, so people know to expect this behaviour.